### PR TITLE
Build cleanup

### PR DESCRIPTION
**Pull request by jrummel**
Closes issue #365  closes issue #59

### CODE REVIEW

File: build-support/emp3-gradle-plugin/src/main/resources/android-app.gradle [hunk 1]
```diff
@@ -37,3 +38,10 @@ android {
         abortOnError false
     }
 }
+
+task testMe {
```
Comment (stevendlander) [inline build-support/emp3-gradle-plugin/src/main/resources/android-app.gradle]: Is this for debug purposes or should it remain in this file?

File: gradle/signAndVerify.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# $4 - Store password
+# $5 - Key password
+# $6 - Full path to desired name of release apk.
+# $7 - $ANDROID_HOME
+jarsigner -verbose -sigalg SHA1withRSA -digestalg SHA1 -keystore $1/emp3_keystore.jks -storepass $4 -keypass $5 $2 emp_release_store
+jarsigner -verify $2
+$7/build-tools/26.0.2/zipalign -v 4 $2 $6
+cd $3
+$1/gradlew releaseChecksum
+cd $1
```
Comment (stevendlander) [inline gradle/signAndVerify.sh]: This shouldn't be necessary, bash scripts should put you back in the current working directory when the script has finished executing.

**Review by stevendlander — CHANGES_REQUESTED** [review]

File: build-support/emp3-gradle-plugin/src/main/resources/android-app.gradle (hunk 1 above)
Comment (jrummel) [reply build-support/emp3-gradle-plugin/src/main/resources/android-app.gradle]: Whoops, thats for debug purposes

File: gradle/signAndVerify.sh (hunk 2 above)
Comment (jrummel) [reply gradle/signAndVerify.sh]: Removed

**Review by stevendlander — APPROVED** [review]
Thank you

**Pull request merged**